### DISCUSSION TITLE
Salesforce也大裁員

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#663] [General]
Salesforce 在去年12月的時候僱用了超過79,000位員工，而在這星期三的時候卻宣布要裁員10%的員工。
Salesforce co-CEO Marc Benioff説：
“As our revenue accelerated through the pandemic, we hired too many people leading into this economic downturn we’re now facing, and I take responsibility for that.”
分析家Brent Bracelin説Salesforce這次裁員每年可以減少1.5billion的花費且公司開發的邊際效益可以從21%增加到26％
由於收入成長緩滿且經濟不景氣，Salesforce與各大科技公司一樣，都開始大量裁員，真的好希望我們畢業的時候，景氣可以回升，不然就只能乖乖讀研究所了。

文章來源：https://www.cnbc.com/2023/01/04/salesforce-is-cutting-10percent-of-its-workforce-more-than-7000-employees.html

**Comment by Yilin20021016** [comment]
太可怕了 我為我的未來擔憂
